plugins/permission-node/api-report.md

```diff
@@ -0,0 +1,139 @@
+## API Report File for "@backstage/plugin-permission-node"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AuthorizeRequest } from '@backstage/plugin-permission-common';
+import { AuthorizeResult } from '@backstage/plugin-permission-common';
+import { BackstageIdentity } from '@backstage/plugin-auth-backend';
+import { PermissionCondition } from '@backstage/plugin-permission-common';
+import { PermissionCriteria } from '@backstage/plugin-permission-common';
+import { Router } from 'express-serve-static-core';
```

Comment:
To investigate: why is Router now imported from express-serve-static-core in the api-report?

Reply:
It was because we weren't explicitly declaring the return type of `createPermissionIntegrationRouter`, TS was inferring it as `express-serve-static-core#Router` instead of `express#Router` (a perfectly valid choice, they're the same type). Fixed by explicitly declaring the return type as `express#Router`.